Core/Core/Store/Store.swift

```diff
@@ -73,6 +73,19 @@ public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate {
     public private(set) var requested: Bool = false
     public private(set) var error: Error?
 
+    // The default implementation of objectWillChange requires at least one
+    // @Published property, but we can't have stored properties that are iOS 13+
+    private var _objectWillChange: Any?
+    @available(iOSApplicationExtension 13.0, *)
+    public var objectWillChange: ObservableObjectPublisher {
+        if let publisher = _objectWillChange as? ObservableObjectPublisher {
+            return publisher
+        }
+        let publisher = ObservableObjectPublisher()
+        _objectWillChange = publisher
+        return publisher
+    }
+
```

Comment:
Implementing this was the secret to getting ObservableObject working properly

Reply:
nice. Looks like we can just make some things `@Published` when we drop iOS 12, and we won't need this anymore, right?

Reply:
Yeah, I think so